src/main/java/com/example/tweet/dao/TweetDao.java

```diff
@@ -47,4 +49,19 @@ public int deleteOneTweet(int id) {
         int resultNum = cnn.update(sql, id);
         return resultNum;
     }
+
+    public Tweet selectTweetById(int tweetId) {
```

Comment:
```suggestion
    public Tweet selectTweetById(final int tweetId) {
```
宗教的理由により、引数は不変にして頂けると。。

Reply:
気が向いた時にまとめて対応します。

Reply:
引数が
`int`などの値であれば、変更ができなくなるが、そもそも変更出来たところで副作用が発生しない。
`Map`や`List`のようなオブジェクトであれば、要素自体は変更でき、副作用から守り切れない。
といった感じで、`final`を付ける意味がいまいちピンとこないのです。
Java Silverの範囲だから、入れたい感じでしょうか？

Reply:
会話済